doc/backend/norms-everkuendung-api-v1.yaml

```diff
@@ -0,0 +1,153 @@
+
+servers:
+  - url: /api/v1
+    description: Backend
+
+paths:
+  /verkuendungen:
+    post:
+      summary: Annahme von Verkündungen.
+      description: |
+        Die Schnittstelle erlaubt das Hochladen eines Normendokumentationspaketes bis zu einer Größe von 200mb. Das Dokumentationspaket liegt in Form eines ZIP-Archivs vor (`bgbl{1|2}_{jahr}_{nr}.zip`). Es enthält folgende Daten:
+
+        - Rechtsetzungsdokument (XML, PDF)
+        - Regelungstext (XML)
+        - soweit vorhanden: Strukturierte Anlage(n) (XML)
+        - soweit vorhanden: Nicht-strukturierte Anlagen, z.B. eingebundene Bilder, PDF-Formulare (etwa: PDF, JPG, PNG)
+        - optional: konsolidierte Fassung aus der E-Gesetzgebung (PDF)
+
+        Das Archiv wird während der Anfrage innerhalb von 120 Sekunden validiert. Eine erfolgreiche Übermittelung wird mit dem Status-Code `200 OK` bestätigt.
+
+        Wenn die Validierung fehlschlägt oder nicht innerhalb dieser Zeit abgeschlossen werden kann, wird die Anfrage mit einem Fehler beendet. Falls der Fehler aufseiten von NeuRIS liegt, sollte die Übertragung nach einem gewissen Zeitraum ernet versucht werden. Dieser Zeitraum kann dem `Retry-After`-Header der Antwort entnommen werden. Wurde die Verkündung nach 5 Versuchen nicht angenommen, muss ein manueller Austausch mit der Normendokumentation stattfinden.
+
+        ## Offene Fragen
+
+        - Die Dateiformate für nicht-strukturierte Anlagen sind noch nicht abschließend geklärt.
+        - Es ist noch offen, welchen Mehrwert die konsolidierte Fassung bringt und ob sich die Übermittelung lohnt.
+        - Art und Format der Signatur müssen noch definiert werden.
+        - Es muss entschieden werden, ob nur das ZIP-Archiv und/oder die Dateien im Archiv signiert werden.
+        - Das Verfahren zum Austausch des öffentlichen Schlüssels für die Verifizierung der Signatur muss noch definiert werden.
+        - Das Verfahren zum manuellen Übertragen der Daten, wenn die Verkündung wiederholt nicht angenommen wurde, muss noch definiert werden.
+        - Sollte die Antwort auf eine erfolgreiche Anfrage weitere Informationen enthalten oder reicht der Statuscode?
+        - Das Vorgehen und ggf. die Schnittstelle für die Übergangsphase, d.h. bevor die E-Gesetzgebung Daten aus dem LEA-Editor liefert, muss noch definiert werden.
+        - Eventuell wird es in Zukunft notwendig sein, statt einer synchronen Anfrage eine asynchrone Anfrage zu verwenden, damit das Backend größere Datenmengen besser verarbeiten kann. Es ist zu klären, ob wir die Schnittstelle direkt so gestalten wollen, oder ob das eine künftige Ausbaustufe sein wird.
+      security:
```

Comment:
Should we apply the security at the root level (for all endpoints to come, if some new come) instead of at the endpoint level?